packages/compose/src/lazy-load/index.js

```diff
@@ -0,0 +1,78 @@
+import { Component } from '@wordpress/element';
```

Comment:
This higher-order component has been added to the `compose` package for now. I was wondering if it should be its own package.

Reply:
Should it be part of the Babel transform package? If it's specifically meant for handling asynchronous components, including the component loader doesn't seem too farfetched.